grommet/grommet
# Fix #5680

Change type definitions to match documentation.

#### What does this PR do?

Update type definitions for TypeScript projects that use Grommet.

#### Where should the reviewer start?

There is only one file that has been changed. 

#### What testing has been done on this PR?

The builds have run successfully, as well as lint and test scripts in **package.json**

#### How should this be manually tested?

Create a TypeScript React project and add a `<Button>` similar to the one shown in #5680. The project should compile 
successfully.

#### What are the relevant issues?

#5680 

#### Do the grommet docs need to be updated?

No. This is a code change to match the existing documentation.

#### Should this PR be mentioned in the release notes?

Possible.

#### Is this change backwards compatible or is it a breaking change?

It is not a breaking change.
